modules/nextflow/src/test/groovy/nextflow/cli/CmdInfoTest.groovy

```diff
@@ -89,7 +89,7 @@ class CmdInfoTest extends Specification {
         json.repository == "https://github.com/nextflow-io/hello"
         json.localPath == "$tempDir/nextflow-io/hello"
         json.manifest.mainScript == 'main.nf'
-        json.manifest.defaultBranch == 'master'
+        json.manifest.defaultBranch == null
```

Comment:
Not getting this

Reply:
The `Manifest` object now used to return `master` if `defaultBranch` isn't specified in the manifest file. 

Now it returns `null` so the `AssetManager` can tell the difference between not-set-in-manifest vs explicitly-set-to-master.